webapp/graphite/render/functions.py

```diff
@@ -983,11 +983,9 @@ def movingAverage(requestContext, seriesList, windowSize):
 
   return result
 
-def cumulative(requestContext, seriesList, consolidationFunc='sum'):
+def cumulative(requestContext, seriesList):
   """
-  Takes one metric or a wildcard seriesList, and an optional function.
-
-  Valid functions are 'sum', 'average', 'min', and 'max'
+  Takes one metric or a wildcard seriesList
 
   Sets the consolidation function to 'sum' for the given metric seriesList.
```

Comment:
Do you think we should expand on this for people who might not read ahead to what `consolidateBy` does?